urdfenvs/urdf_common/reward.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import List
+from urdfenvs.sensors.sensor import Sensor
+
+
+class Reward(ABC):
+    @abstractmethod
+    # TODO: Get a clear list of things that could be needed for calculating reward.
+    def calculateReward(self, observation) -> float: pass
```

Comment:
@behradkhadem I changed it using the observation directly as it is easier to work with, than with the sensor.

Reply:
TBH, I thought about the same thing at first, but I thought it would be cleaner to use sensors (and maybe cameras) directly. Do we have access to every sensor and camera inside `observation` variable? (I haven't used the camera in this package.)

Reply:
I agree with that. I haven't tested cameras yet in the environment. However, everything you would use to calculate your reward should also be accessible for an RL agent using the observation, right?

Reply:
Well, I guess so. Let me try using the package for a real RL problem and see whether there are any problems or not.

Reply:
Sounds like a good plan. I am really curious to hear about your experience.